text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/resolver/TagResolver.java

```diff
@@ -142,6 +142,56 @@ public boolean has(final @NotNull String name) {
     };
   }
 
+  /**
+   * Create a tag resolver that only responds to certain tag names, and whose value does not depend on that name.
+   *
+   * <p>
+   * This method creates a special resolver which listens to tags with named arguments instead of sequences ones.
```

Comment:
```suggestion
   * This method creates a special resolver which listens to tags with named arguments instead of sequential ones.
```
there looks to be a couple other spots with the same issue

Reply:
I think I fixed them all